README.md

```diff
@@ -1,65 +1,76 @@
 # ICICLE
 
-<div align="center">ICICLE is a high-performance cryptographic acceleration library designed to optimize cryptographic computations across various hardware platforms, including CPUs, GPUs, and other accelerators.</div>
+<div align="center">
+  ICICLE is a high-performance cryptographic acceleration library designed to optimize cryptographic computations across various hardware platforms, including CPUs, GPUs, and other accelerators.
+</div>
 
 <p align="center">
-  <img alt="ICICLE" width="300" height="300" src="https://user-images.githubusercontent.com/2446179/223707486-ed8eb5ab-0616-4601-8557-12050df8ccf7.png"/>
+  <br>
+  <img alt="ICICLE" width="300" height="300" src="McPaw.gif"/>
 </p>
+
 <p align="center">
+  <br>
+  <br>
   <a href="https://discord.gg/EVVXTdt6DF">
-    <img src="https://img.shields.io/discord/1063033227788423299?logo=discord" alt="Chat with us on Discord">
+    <img src="https://img.shields.io/discord/1063033227788423299?logo=discord&style=flat&labelColor=black" alt="Chat with us on Discord">
   </a>
-  <a href="https://twitter.com/intent/follow?screen_name=Ingo_zk">
-    <img src="https://img.shields.io/twitter/follow/Ingo_zk?style=social&logo=twitter" alt="Follow us on Twitter">
+  <a href="https://www.linkedin.com/company/ingonyama">
+    <img src="https://img.shields.io/badge/follow-LinkedIn-blue?style=flat&logo=linkedin&labelColor=black" alt="Follow us on LinkedIn">
   </a>
   <a href="https://github.com/ingonyama-zk/icicle/releases">
-    <img src="https://img.shields.io/github/v/release/ingonyama-zk/icicle" alt="GitHub Release">
+    <img src="https://img.shields.io/badge/release-v2.8.0-lightblue?style=flat&labelColor=black" alt="GitHub Release">
```

Comment:
We will need to manually update the version number here on each new version:
I think it would be better to use the same URL we had before which auto updates with the style added:
```suggestion
    <img src="https://img.shields.io/github/v/release/ingonyama-zk/icicle?style=flat&labelColor=black&color=lightblue" alt="GitHub Release">
```